data/deep missions.txt

```diff
@@ -0,0 +1,1121 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Given the current skeleton, the following would need to be done in order to unlock the Bactrian:
+#	10 mystery cube jobs
+#	3 transport scientist jobs
+#	3 escort science vessel jobs
+#	3 record academic conference jobs
+#	2 convoy missions
+#	The current TMBR string + the 3 missions added by this
+#	The Deep: Mystery Cubes string added by this
+#	The Deep Archaeology string
+#	The Deep: Singularity string added by this
+#	The Deep: Remnant string added by this
+#	The Remnant intro missions (kind of optional, as the player may choose to plunder Remnant tech, but this would ruin future storylines for the player; do you really want the Bactrian that much?)
+#	The Deep: Star Queen Rescue string added by this
+
+
+###########################################################
+#    The following is a skeleton for different mission    #
+# that will be filled in once the storyboard is finalized #
+###########################################################
+
+# Only need to do 2 of 3 convoy missions
+# Escort convoys of freighters to and from different regions of space to gather materials for experiments that the Deep is conducting.
+mission "Deep: Syndicate Convoy" # Korath-based tech
+	minor
+	name `Pickup from Hephaestus`
+	description `Escort a convoy of freighters to <stopovers> where they will pick up materials for scientific experiments. Then return the convoy to <destination> for a payment of <payment>.`
+	source
+		attributes deep
+	stopovers Hephaestus
+	to offer
+		random < 30
+		"mystery cube" >= 10
+		"Transport scientists: done" >= 3
+		"Escort science vessel: done" >= 3
+		"Record academic conference: done" >= 3
+	on offer
+		conversation
+			``
+	
+	npc accompany save
+		personality timid escort
+		government Merchant
+		fleet
+			names civilian
+			variant
+				"Mule"
+				"Bulk Freighter" 2
+	npc
+		personality nemesis waiting
+		system
+			distance 3
```

Comment:
I get what you're going for here, but I wonder if there is a better way to set it up. There's a pretty good chance these NPC fleets don't spawn in the desired systems. (e.g. pick chokepoints).

Of course, if tehhowch/endless-sky#3 merges, then these could be patrolling and the problem is solved